docs/content/connectors/redis.md

```diff
@@ -0,0 +1,78 @@
+# Redis
+
+FeatHub provides `RedisSource` to read data from a Redis database and
+`RedisSink` to materialize feature view to a Redis database. Currently,
+`RedisSource` can only be used as an online store source.
+
+## Supported Processors and Usages
+
+- Flink: Lookup<sup>1</sup>, Streaming Upsert
+
+1. Only supported in OnDemandFeatureView currently.
+
+## Examples
+
+Here are the examples of using `RedisSource` and `RedisSink`:
+
+### Use as Streaming Upsert Sink
+
+ `RedisSink` works in upsert mode and requires that the feature view to be
+ materialized must have keys.
+
+```python
+feature_view = DerivedFeatureView(...)
+
+sink = RedisSink(
+    host="host",
+    port="port",
+    username="username",
+    password="password",
+    db_num=0,
+    namespace="namespace",
+)
+
+featub_client.materialize_features(
+    features=feature_view,
+    sink=sink,
+    allow_overwrite=True,
+).wait(30000)
+```
+
+### Use as Lookup Source for OnDemandFeatureView
+
+```python
+feature_table_schema = (
+    Schema.new_builder()
+    .column("id", types.Int64)
+    .column("feature_1", types.Int64)
+    .column("ts", types.String)
+    .build()
+)
+
+source = RedisSource(
+    name="feature_table",
+    host="host",
+    port=int("port_num"),
```

Comment:
Maybe easier to understand if we put the default port number of Redis, or we can remove the `port` argument as it is optional.